module/VuFind/src/VuFind/AjaxHandler/GetSearchResults.php

```diff
@@ -219,7 +219,6 @@ protected function renderResults(Params $params, Results $results): ?string
         if ('search' !== $baseAction && !$this->renderer->resolver($templatePath)) {
             $templatePath = 'search/results-list.phtml';
         }
-        $params = $results->getParams();
```

Comment:
I wonder if it's worth aliasing the Params class as ParamsHelper or QueryParams to make this less potentially confusing -- had to read the code carefully to understand this change (which does make perfect sense upon examination). Maybe better done as a follow-up action, if it's worth doing it all, since it's not really related to the rest of the work here. Just an observation!

Reply:
I see you've renamed the parameter instead of the type, which does the job just as well -- thanks!

Reply:
I decided to rename both the Params and the actual parameter variable for clarity. Ended up with ParamsHelper since QueryParams could be confused to params in the query string.